platform-tests/src/test/java/org/junit/platform/engine/support/hierarchical/ThrowableCollectorTests.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.engine.support.hierarchical;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.platform.engine.TestExecutionResult.Status.ABORTED;
+import static org.junit.platform.engine.TestExecutionResult.Status.FAILED;
+import static org.junit.platform.engine.TestExecutionResult.Status.SUCCESSFUL;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.engine.TestExecutionResult;
+
+/**
+ * @since 1.6
+ */
+class ThrowableCollectorTests {
+
+	@Test
+	void executeSuccessful() {
```

Comment:
Maybe `successfulExecution` or `executeSuccessfully`?

Reply:
Indeed: `successfulExecution()`, `abortedExecution()` and `failedExecution()`

Reply:
These test method names came from `SingleThreadExecutorTests`. I just changed the `executeSafely` prefix to `execute`. I'm happy to change them; I was just trying to keep it consistent.